risc0/zkvm/src/host/server/prove/tests.rs

```diff
@@ -498,96 +498,81 @@ fn sys_input() {
 #[cfg(feature = "docker")]
 mod docker {
     use crate::{
-        get_prover_server, recursion::identity_p254, CompactReceipt, ExecutorEnv, ExecutorImpl,
-        InnerReceipt, ProverOpts, Receipt, ReceiptKind, VerifierContext,
+        get_prover_server,
+        host::server::prove::{DevModeProver, ProverServer},
+        ExecutorEnv, InnerReceipt, ProverOpts, Receipt, ReceiptKind,
     };
-    use anyhow::{bail, Result};
-    use risc0_groth16::docker::stark_to_snark;
     use risc0_zkvm_methods::{multi_test::MultiTestSpec, MULTI_TEST_ELF, MULTI_TEST_ID};
 
     #[test]
-    fn stark2snark() {
+    fn prove_nothing_compact() {
```

Comment:
I'd much prefer to keep the old name, it's what i use for filtering tests

Reply:
Changed back in 9af4758c18320d63188a75d9063c85a391b9666e